### PR TITLE
feat(desktop/wallet) Adding Setttings tab in wallet 2

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
https://github.com/status-im/StatusQ/pull/370

StatusQ PR link : https://github.com/status-im/StatusQ/pull/388

### CODE REVIEW

File: ui/app/AppLayouts/WalletV2/SettingsTab.qml [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+            anchors.leftMargin: 20
+            anchors.right: parent.right
+            anchors.rightMargin: 20
+
+            visible : walletV2Model.accountsView.currentAccount.walletType === Constants.keyWalletType
+            expandable: true
+            icon.name: "seed-phrase"
+            primaryText: qsTr("View private key")
+            secondaryText: qsTr("Back up your seed phrase now to secure this account")
+            button.text: qsTr("View private key")
```
Comment (alaibe) [inline ui/app/AppLayouts/WalletV2/SettingsTab.qml:65]: Is the secondary text correct? It seems this text should be with: Back up seed phrase not with view private key?

File: src/app/wallet/v2/views/utils.nim [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import atomics, strformat, strutils, sequtils, json, std/wrapnils, parseUtils, tables, chronicles, web3/[ethtypes, conversions], stint
+import NimQml, json, sequtils, chronicles, strutils, strformat, json
+
+logScope:
+  topics = "utils-view"
```
Comment (alaibe) [inline src/app/wallet/v2/views/utils.nim:5]: nit: align naming between qml and nim: Use settings in both cases?

**Review by alaibe — APPROVED** [review]
LGTM but pls have a look to comment below

File: ui/app/AppLayouts/WalletV2/SettingsTab.qml (hunk 1 above)
Comment (Khushboo-dev-cpp) [reply ui/app/AppLayouts/WalletV2/SettingsTab.qml:65]: You are right. This text doesn't make sense will check with design and update it!

File: src/app/wallet/v2/views/utils.nim (hunk 2 above)
Comment (Khushboo-dev-cpp) [reply src/app/wallet/v2/views/utils.nim:5]: hmmm 🧐 makes sense!

Comment (Khushboo-dev-cpp) [reply src/app/wallet/v2/views/utils.nim]: Done

File: ui/app/AppLayouts/WalletV2/SettingsTab.qml (hunk 1 above)
Comment (Khushboo-dev-cpp) [reply ui/app/AppLayouts/WalletV2/SettingsTab.qml:65]: I spoke to Carl and he mentioned this should be changes soon. He mentioned we can make it "view your seed phrase and ensure it's stored in a safe place" and it is available in 2 cases when account I imported using a PK and using a Seed Phrase

**Pull request merged**